README.md

```diff
@@ -62,6 +62,10 @@ In the end of pull request name must be `#<ZenHub isssue number>`.
 + <a href="https://devdocs.io/html/"><b>HTML</b></a> for website markup
 + <a href="https://devdocs.io/css/"><b>CSS</b></a> for styling website
 + <a href="https://devdocs.io/javascript/"><b>JavaScript</b></a> for webside frontend scripts
++ <a href="https://www.npmjs.com/"><b>Node package manager<b></a> for installing all front-end tools
```

Comment:
*< / b >

Reply:
May be we will use [md syntax](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file) instead of html?

Reply:
> May be we will use [md syntax](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file) instead of html?

It makes sense.